style.css

```
@@ -0,0 +1,1381 @@
+.textcontainer {
+  display: flex;
+  flex-direction: column;
+}
+
+.imagecontainer {
+  display: flex;
+  justify-content: space-between;
+}
+
+.defineimage {
+  width: 111px;
+  height: 74px;
```

[OPTIONAL]

- Perhaps, in the future, please consider using fluid measurements likes rem. 1 rem is the same as 16px. They are good because they help with responsive design.
```suggestion
  width: 8rem;
  height: 4.5rem;
```

---

Thank you very much for your valuable suggestion. I will definitely incorporate in my future projects since this is optional comment.